### ISSUE TITLE
Unnecessary ARIA labels

### ISSUE THREAD

**Issue by oliverjam** [fac22/week2-AAMJ#32]
https://github.com/fac22/week2-AAMJ/blob/8c9da43a5e755ac6d8a7b06ada9aa185954002a4/routes/form.js#L31

You don't _have_ to use `aria-label` for buttons. It's generally so you can provide a text description of an action that is only communicated visually. E.g. a button containing just an icon is meaningless if you cannot see the icon. However your buttons already have text labels, so it's a little confusing to override these with another label.

It's generally best to try and provide the _same_ experience to users of assistive tech like screen readers—if the word "Send" is a good enough label for a sighted user it should be good enough for a partially-sighted one. If you think "Submit your details" is a better description of what the button does then make that the normal text label for everybody.
